parquet-thrift/src/main/java/parquet/thrift/ThriftSchemaConvertVisitor.java

```diff
@@ -56,17 +58,19 @@
  */
 public class ThriftSchemaConvertVisitor implements ThriftType.TypeVisitor {
 
-  public FieldProjectionFilter getFieldProjectionFilter() {
-    return fieldProjectionFilter;
-  }
+    private static final Log logger = Log.getLog(ThriftSchemaConvertVisitor.class);
```

Comment:
Looks like your indentation was accidentally off, causing this change to be larger than you intended.